starboard/android/apk/app/src/main/java/dev/cobalt/media/VideoFrameReleaseTimeHelper.java

```diff
@@ -118,7 +120,17 @@ public void disable() {
    */
   @SuppressWarnings("unused")
   @UsedByNative
-  public long adjustReleaseTime(long framePresentationTimeUs, long unadjustedReleaseTimeNs) {
+  public long adjustReleaseTime(
+      long framePresentationTimeUs, long unadjustedReleaseTimeNs, double playbackRate) {
+    if (playbackRate == 0) {
+      return unadjustedReleaseTimeNs;
+    }
+    if (playbackRate != lastPlaybackRate) {
+      // Resync if playback rate has changed.
+      haveSync = false;
```

Comment:
Resyncing during any playback rate changes probably shouldn't be an issue in production, but just to bring the following up:
`haveSync` serves two purposes when it's set to true:
1. Re-collect the time bases.
2. Wait another MIN_FRAMES_FOR_ADJUSTMENT (6) frames to re-establish average frame duration.
When playback rate has been changed, we should redo 1.  Redoing 2 leads to 6 unadjusted frames, and we should be able to figure out the average frame duration without waiting for another 6 frames.

Reply:
I think it should be ok. The unadjusted release time sometimes might have dropped frame due to the cadence, but it should only have minor impact in production.